lib/instance.js

```diff
@@ -245,6 +246,29 @@ class Instance {
         this.system.setEnvironment(env === 'development', setNodeEnv);
     }
 
+    /**
+     * Gets a list of availble configurations for this instance
+     *
+     * @method getAvailableConfigs
+     * @return {Promise<object>}
```

Comment:
```suggestion
     * @return {Promise<{[s: string]: Config}>}
```

I _think_ this should work as valid jsdoc

Reply:
yup - thanks for the suggestion 😄